docs/linux.md

```diff
@@ -178,7 +178,10 @@ Remove the ollama binary from your bin directory (either `/usr/local/bin`, `/usr
 ```shell
 sudo rm $(which ollama)
 ```
-
+Remove the ollama libraries from your lib directory (`/usr/lib`):
+```shell
+sudo rm /usr/lib/ollama
```

Comment:
Depending on the system, and if it was manually installed or installed with the official [install script](https://github.com/ollama/ollama/blob/main/scripts/install.sh#L69-L71), the location may vary.  The step above should locate ollama, so perhaps we should adjust the sequence to leverage that.  Something like the following as a step **before** removing the main executable:
```shell
sudo rm -r $(dirname $(which ollama))/../lib/ollama/
```
(also needs the `-r` to handle directories.)